From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Dolphin XD <dolphinxd@users.noreply.github.com>
Date: Thu, 8 Oct 2026 01:55:08 +0000
Subject: [PATCH 3/3] Theme Dolphin's screens in Dolphin XD's colors

Dark mode (Dolphin XD's default) uses purple, red and black instead of
Dolphin's blue, so the settings screens and in-game menu match the XD
main menu.
---
 .../main/res/values-night/dolphin_colors.xml  | 46 +++++++++----------
 1 file changed, 23 insertions(+), 23 deletions(-)

diff --git a/Source/Android/app/src/main/res/values-night/dolphin_colors.xml b/Source/Android/app/src/main/res/values-night/dolphin_colors.xml
index 39e5ccc..bc0b510 100644
--- a/Source/Android/app/src/main/res/values-night/dolphin_colors.xml
+++ b/Source/Android/app/src/main/res/values-night/dolphin_colors.xml
@@ -1,31 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- Generated by https://material-foundation.github.io/material-theme-builder/ -->
-    <color name="dolphin_primary">#9ECAFF</color>
-    <color name="dolphin_onPrimary">#003258</color>
-    <color name="dolphin_primaryContainer">#00497D</color>
-    <color name="dolphin_onPrimaryContainer">#D1E4FF</color>
-    <color name="dolphin_secondary">#BBC7DB</color>
-    <color name="dolphin_onSecondary">#253140</color>
-    <color name="dolphin_secondaryContainer">#3B4858</color>
-    <color name="dolphin_onSecondaryContainer">#D7E3F7</color>
-    <color name="dolphin_tertiary">#D6BEE4</color>
-    <color name="dolphin_onTertiary">#3B2948</color>
-    <color name="dolphin_tertiaryContainer">#523F5F</color>
-    <color name="dolphin_onTertiaryContainer">#F2DAFF</color>
+    <!-- Dolphin XD: Shadow Lugia purple, XD red and black (dark mode is Dolphin XD's default). -->
+    <color name="dolphin_primary">#FF5A62</color>
+    <color name="dolphin_onPrimary">#FFFFFF</color>
+    <color name="dolphin_primaryContainer">#7A0E15</color>
+    <color name="dolphin_onPrimaryContainer">#FFDAD7</color>
+    <color name="dolphin_secondary">#C9B8FF</color>
+    <color name="dolphin_onSecondary">#24124F</color>
+    <color name="dolphin_secondaryContainer">#3B2C6E</color>
+    <color name="dolphin_onSecondaryContainer">#E9E0FF</color>
+    <color name="dolphin_tertiary">#C9B8FF</color>
+    <color name="dolphin_onTertiary">#24124F</color>
+    <color name="dolphin_tertiaryContainer">#3B2C6E</color>
+    <color name="dolphin_onTertiaryContainer">#E9E0FF</color>
     <color name="dolphin_error">#FFB4AB</color>
     <color name="dolphin_errorContainer">#93000A</color>
     <color name="dolphin_onError">#690005</color>
     <color name="dolphin_onErrorContainer">#FFDAD6</color>
-    <color name="dolphin_background">#1A1C1E</color>
-    <color name="dolphin_onBackground">#E2E2E6</color>
-    <color name="dolphin_surface">#1A1C1E</color>
-    <color name="dolphin_onSurface">#E2E2E6</color>
-    <color name="dolphin_surfaceVariant">#43474E</color>
-    <color name="dolphin_onSurfaceVariant">#C3C7CF</color>
-    <color name="dolphin_outline">#8D9199</color>
-    <color name="dolphin_inverseOnSurface">#1A1C1E</color>
-    <color name="dolphin_inverseSurface">#E2E2E6</color>
-    <color name="dolphin_inversePrimary">#0061A4</color>
+    <color name="dolphin_background">#0B0814</color>
+    <color name="dolphin_onBackground">#F4F1FB</color>
+    <color name="dolphin_surface">#0B0814</color>
+    <color name="dolphin_onSurface">#F4F1FB</color>
+    <color name="dolphin_surfaceVariant">#2E2448</color>
+    <color name="dolphin_onSurfaceVariant">#CFC6E6</color>
+    <color name="dolphin_outline">#8D82AE</color>
+    <color name="dolphin_inverseOnSurface">#0B0814</color>
+    <color name="dolphin_inverseSurface">#F4F1FB</color>
+    <color name="dolphin_inversePrimary">#A3141C</color>
     <color name="dolphin_shadow">#000000</color>
 </resources>
